dev-packages/application-package/src/index.ts

```diff
@@ -5,8 +5,7 @@
  * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
  */
 
-export * from './rebuild';
 export * from './npm-registry';
 export * from './extension-package';
 export * from './application-package';
-export * from './application-package-manager';
+export * from './application-process';
```

Comment:
`application-process` is used only in the manager, should go to another package as well